aws/variables.tf

```diff
@@ -33,18 +35,18 @@ variable "login_user" {
 # the name of the ssh key pair in aws
 # this tells aws to provision instances with this keypair
 variable "default_ssh_key" {
-  default = "default-aws-pem.pem"
+  default = "aws_default_key"
 }
 
 # the path to this key pair locally
 # this is used by terrafrom to ssh in and run any provisioning steps
 # NOTE/GOTCHA: on osx this generally goes along the lines '/Users/username/.....' and on linux it is '/home/username/...'
 variable "default_ssh_key_path" {
-  default = "/home/username/.ssh/default-aws-pem.pem"
+  default = "/Users/tarakaprabhu/.ssh/aws_default_key.pem"
```

Comment:
@tarakaprabhuchinta just need to switch this back and should be good mate

Reply:
@joshuafernandes switched it back to default path and updated the code